.github/workflows/lint.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: Analysing the code with pylint
         run: |
-          pylint $(git ls-files '*.py') --fail-under 8
+          pylint $(git ls-files '*.py') --fail-under 7.8
```

Comment:
Can't we increase this mark? 7.95 is not enough?

This indicator should not be the alpha and the omega of the project, however this is a guarantee to continue to develop things in a pretty satisfying direction. We can do exception (i.e. lower the required mark) to introduce big amount of code if we decide the changes are of first importance, however they should be rather limited (in frequency and intensity).

Reply:
ok, fixed at 7.95.